### PR TITLE
hec token creation

### PR DESCRIPTION
**Pull request by srivickynesh**
Created the token via inputs.conf for indexes: rh_rhtap_dev_app and rh_rhtap_dev_audit
Both Token is different and hardcoded since they use localhost instance
Rebased on #1 and added the hec token here.
```
Container from image 'splunk-dev-image' started successfully!
Waiting for Splunk CLI to come up
Splunk CLI is Up
Event sent successfully!
Event sent successfully!
```

### CODE REVIEW

File: run_splunk_container.sh [hunk 1]
```diff
@@ -14,7 +17,11 @@ build_and_run_image() {
     podman build -t "$IMAGE_NAME" . && echo "Image '$IMAGE_NAME' built successfully!" || { echo "Error: Failed to build the image."; exit 1; }
 
     echo "Running Podman container..."
-    podman run -d -e SPLUNK_PASSWORD="$SPLUNK_PASSWORD" -e SPLUNK_START_ARGS="$SPLUNK_START_ARGS" -p 8000:$WEB_PORT -p 8089:$API_PORT "$IMAGE_NAME" && echo "Container from image '$IMAGE_NAME' started successfully!" || { echo "Error: Failed to start the container."; exit 1; }
+    podman run -d -e SPLUNK_PASSWORD="$SPLUNK_PASSWORD" -e SPLUNK_START_ARGS="$SPLUNK_START_ARGS" -p 8000:$WEB_PORT -p 8089:$API_PORT -p 8088:$HEC_PORT "$IMAGE_NAME" && echo "Container from image '$IMAGE_NAME' started successfully!" || { echo "Error: Failed to start the container."; exit 1; }
```
Comment (eisraeli) [inline run_splunk_container.sh]: Please break this line

File: run_splunk_container.sh [hunk 2]
```diff
@@ -23,17 +30,37 @@ wait_for_splunk_cli() {
...
+
+    if [ $? -ne 0 ]; then
+        echo "Curl command failed"
+        return 1
+    fi
+
+    success_code=$(echo "$response" | jq -r '.code')
+
+    if [ "$success_code" == "0" ]; then
+        echo "Event sent successfully!"
```
Comment (eisraeli) [inline run_splunk_container.sh:73]: Print here some more information like the HEC EP & the token.
It's for local use and can be informative instead looking at the code.

Comment (srivickynesh) [reply run_splunk_container.sh:73]: tokens are easily reachable when accessing from UI. If anyone wants to access it. I will add a URL to access the token.

Comment (eisraeli) [reply run_splunk_container.sh:73]: Alright.

**Pull request merged**